aws/ae-environment.yml

```diff
@@ -97,6 +97,3 @@ Resources:
         AvailabilityZones:
           Fn::GetAZs:
             Ref: "AWS::Region"
-
-    EpochElasticIP:
```

Comment:
So the idea is removing this because unused, right?

Reply:
The idea of removing this is otherwise deleting the stack will delete the elastic IP too, we don't want that, but to keep it "forever".